data/src/main/java/io/gnosis/data/backend/GatewayApi.kt

```diff
@@ -15,59 +15,58 @@ import java.util.*
 
 interface GatewayApi {
 
-    @GET("/{chainId}/v1/balances/supported-fiat-codes")
+    @GET("/v1/chains/{chainId}/balances/supported-fiat-codes")
```

Comment:
This endpoint is chain independent. As we have a single common source for fiat conversion rates

Should be `/v1/balances/supported-fiat-codes`